plugins/python-build/bin/python-build

```diff
@@ -1561,15 +1579,38 @@ build_package_mac_openssl() {
 
 # Post-install check that the openssl extension was built.
 build_package_verify_openssl() {
-  "$RUBY_BIN" -e 'begin
-    require "openssl"
-  rescue LoadError
-    $stderr.puts "The Ruby openssl extension was not compiled. Missing the OpenSSL lib?"
-    $stderr.puts "Configure options used:"
-    require "rbconfig"; require "shellwords"
-    RbConfig::CONFIG.fetch("configure_args").shellsplit.each { |arg| $stderr.puts "  #{arg}" }
-    exit 1
-  end' >&4 2>&1
+  "$RUBY_BIN" -e '
```

Comment:
We should do the Python equivalent `import _ssl`, but it can be a separate issue.

Reply:
JFYI, in fact, this is unused code from python-build as we're using `verify_python_module` instead.

https://github.com/pyenv/pyenv/blob/v1.2.11/plugins/python-build/bin/python-build#L1846